### PR TITLE
Update changelog and version for 1.22.3

### PR DESCRIPTION
**Pull request by Colengms**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.22.3: September 16, 2024
+### Enhancement
+* Add support for providing additional context information to Copilot Chat. [PR#12685](https://github.com/microsoft/vscode-cpptools/pull/12685)
+
+### Bug Fixes
+* The compiler path selection control is not in sync with the textbox. [#7427](https://github.com/microsoft/vscode-cpptools/issues/7427)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Normally, we word it like "Fix <the bug>" and not "<the bug>".

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.22.3: September 16, 2024
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Is this date correct? If so, why is the changelog being updated now?

File: Extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+* Add support for providing additional context information to Copilot Chat. [PR#12685](https://github.com/microsoft/vscode-cpptools/pull/12685)
+
+### Bug Fixes
+* The compiler path selection control is not in sync with the textbox. [#7427](https://github.com/microsoft/vscode-cpptools/issues/7427)
+* Fix a string localization issue. [#7824](https://github.com/microsoft/vscode-cpptools/issues/7824) 
+* Stop logging duplicate compiler path messages. [#12445](https://github.com/microsoft/vscode-cpptools/issues/12445)
+* Fix some crashes with recursive includes. [#12643](https://github.com/microsoft/vscode-cpptools/issues/12643)
+* Fix a rare crash on macOS related to `get_memory_usage`. [#12667](https://github.com/microsoft/vscode-cpptools/issues/12667)
+* Fix an issue with Extract to Function formatting. [#12677](https://github.com/microsoft/vscode-cpptools/issues/12677)
+
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md:15]: You forgot https://github.com/microsoft/vscode-cpptools/issues/12690

File: Extension/CHANGELOG.md (hunk 2 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: i.e. can we release Thursday or did we want to wait till Monday for some reason?

File: Extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+* Add support for providing additional context information to Copilot Chat. [PR#12685](https://github.com/microsoft/vscode-cpptools/pull/12685)
+
+### Bug Fixes
+* Fix the compiler selection control not keeping the list in sync with contents of the textbox. [#7427](https://github.com/microsoft/vscode-cpptools/issues/7427)
+* Fix a string localization issue. [#7824](https://github.com/microsoft/vscode-cpptools/issues/7824) 
+* Stop logging duplicate compiler path messages. [#12445](https://github.com/microsoft/vscode-cpptools/issues/12445)
+* Fix some crashes with recursive includes. [#12643](https://github.com/microsoft/vscode-cpptools/issues/12643)
+* Fix a rare crash on macOS related to `get_memory_usage`. [#12667](https://github.com/microsoft/vscode-cpptools/issues/12667)
+* Fix an issue with Extract to Function formatting. [#12677](https://github.com/microsoft/vscode-cpptools/issues/12677)
+* Fix a potential deadlock in `process_paths`. [12690](https://github.com/microsoft/vscode-cpptools/issues/12690)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: `[12690]`->`[#12690]`

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.22.3: September 12, 2024
+### Enhancement
+* Add support for providing additional context information to Copilot Chat. [PR#12685](https://github.com/microsoft/vscode-cpptools/pull/12685)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: `[PR#12685]`->`[PR #12685]`

File: Extension/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,18 @@
...
+
+### Bug Fixes
+* Fix the compiler selection control not keeping the list in sync with contents of the textbox. [#7427](https://github.com/microsoft/vscode-cpptools/issues/7427)
+* Fix a string localization issue. [#7824](https://github.com/microsoft/vscode-cpptools/issues/7824)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md:9]: 7824 should sort before 7427.

File: Extension/CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,18 @@
...
+### Enhancement
+* Add support for providing additional context information to Copilot Chat. [PR#12685](https://github.com/microsoft/vscode-cpptools/pull/12685)
+
+### Bug Fixes
+* Fix the compiler selection control not keeping the list in sync with contents of the textbox. [#7427](https://github.com/microsoft/vscode-cpptools/issues/7427)
+* Fix a string localization issue. [#7824](https://github.com/microsoft/vscode-cpptools/issues/7824) 
+* Stop logging duplicate compiler path messages. [#12445](https://github.com/microsoft/vscode-cpptools/issues/12445)
+* Fix some crashes with recursive includes. [#12643](https://github.com/microsoft/vscode-cpptools/issues/12643)
+* Fix a rare crash on macOS related to `get_memory_usage`. [#12667](https://github.com/microsoft/vscode-cpptools/issues/12667)
+* Fix an issue with Extract to Function formatting. [#12677](https://github.com/microsoft/vscode-cpptools/issues/12677)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: `Extract to Function`->`'Extract to Function'`.

File: Extension/CHANGELOG.md (hunk 6 above)
Comment (Colengms) [reply Extension/CHANGELOG.md:9]: Why should a larger number come before a smaller number?  The others are least to greatest.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**